packages/xrpl/src/models/common/index.ts

```diff
@@ -61,10 +61,31 @@ interface PathStep {
 
 export type Path = PathStep[]
 
+/**
+ * The object that describes the signer in SignerEntries.
+ */
 export interface SignerEntry {
+  /**
+   * The object that describes the signer in SignerEntries.
+   */
   SignerEntry: {
+    /**
+     * An XRP Ledger address whose signature contributes to the multi-signature.
+     * It does not need to be a funded address in the ledger.
+     */
     Account: string
+    /**
+     * The weight of a signature from this signer.
+     * A multi-signature is only valid if the sum weight of the signatures provided meets
+     * or exceeds the signer list's SignerQuorum value.
+     */
     SignerWeight: number
+    /**
+     * An arbitrary 256-bit (32-byte) field that can be used to identify the signer, which
+     * may be useful for smart contracts, or for identifying who controls a key in a large
+     * organization.
+     */
+    WalletLocator?: string
```

Comment:
I think something like `SignerMetadata` would be a better name here because it allows arbitrary data. (And although we think the most common use case will be for identifying the signer, I don't think `WalletLocator` accurately conveys the dynamic data type)

Reply:
@JST5000 `WalletLocator` is the field name that the XRPL expects in the transaction - https://xrpl.org/signerlist.html#signer-entry-object

Reply:
Ah that makes sense, my bad :)